spinalcordtoolbox/deepseg/models.py

```diff
@@ -237,7 +264,25 @@
                              'and Dimitry Van De Ville of EPFL, with the files consisting of lumbar T2w scans (and '
                              'manual spinal cord segmentations) of 11 healthy (non-pathological) patients.',
          'url': 'https://github.com/ivadomed/lumbar_seg_EPFL',
-         'models': ['model_seg_epfl_t2w_lumbar_sc']}
+         'models': ['model_seg_epfl_t2w_lumbar_sc']},
+    'seg_sc_contrast_agnostic':
+        {'description': 'Spinal cord segmentation that is agnostic to contrast',
+         'long_description': 'This segmentation model for contrast agnostic spinal cord segmentation uses an NNUnet '
+                             'architecture, and was created with the MONAI package. Training data was taken from the '
+                             'Spine Generic Multi Subject dataset, with the 6 different contrasts used spanning 267 '
+                             'different healthy (non-pathological) patients.',
+         'url': 'https://github.com/sct-pipeline/contrast-agnostic-softseg-spinalcord/',
+         'models': ['model_seg_sc_contrast_agnostic_softseg_nnunet']},
+    'seg_sc_lesion_t2w_sci':
+        {'description': 'Traumatic SCI spinal cord/lesion seg for T2w contrast',
+         'long_description': 'TBD.',
+         'url': 'https://github.com/ivadomed/model_seg_sci',
+         'models': ['model_seg_sci_multiclass_sc_lesion_nnunet']},
+    'seg_spinal_rootlets_t2w':
+        {'description': 'Segmentation of spinal nerve rootlets for T2w contrast',
+         'long_description': 'TBD.',
```

Comment:
Note to self: Remember to write a long description for the SCI model before merging.